<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainForm</class>
 <widget class="QWidget" name="MainForm">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>887</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(72, 72, 72)</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="titleLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>285</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>285</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/Resources/USDattr.png&quot; width=&quot;45&quot; height=&quot;45&quot; style=&quot;float: left;&quot;/&gt;&lt;font color=&quot;#1F91D0&quot;&gt;&lt;font size=&quot;5&quot;&gt;ASSETLIB BROWSER&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QRadioButton" name="db1RadioButton">
       <property name="styleSheet">
        <string notr="true">color:white; font:bold 14px;</string>
       </property>
       <property name="text">
        <string>ASSETLIB</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="db2RadioButton">
       <property name="styleSheet">
        <string notr="true">color:white; font:bold 14px;</string>
       </property>
       <property name="text">
        <string>ASSETLIB2</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="userNameLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color:white; font:bold 20px;</string>
       </property>
       <property name="text">
        <string>daeseok.chae</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="search_combobox">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QComboBox {color:#cccccc;}
QComboBox::item:selected {
	background-color: #2a2d2e;
}</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="searchEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:white; background-color:#383838</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string>Search Assets</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="topMargin">
      <number>12</number>
     </property>
     <item>
      <widget class="QWidget" name="widget_category" native="true">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="margin" stdset="0">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #cccccc; padding-left: 5;</string>
            </property>
            <property name="text">
             <string>CATEGORY</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="ClickableQLabel" name="update_category_lbl">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Reload</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="../resources.qrc">:/Resources/icon/reload.png</pixmap>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ClickableQLabel" name="add_category_lbl">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Add Category</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="../resources.qrc">:/Resources/icon/plus-2-16.png</pixmap>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="CategoryTreeWidget" name="category_tree_widget">
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QTreeView {color: #C2C2C2; font: bold 14px; width: 100px; background-color: #383838; border: 1px solid #3d3d3d;}
QTreeView::item:selected {background-color: #53728e;color: #C2C2C2;}
QTreeView::item:hover {border: 1px solid #53728e;}
QTreeView:verticalScrollBar {background-color: #FFA91D;alternate-background-color: #FF0000;}</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="headerHidden">
           <bool>true</bool>
          </property>
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_item" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>12</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>12</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #cccccc; padding-left: 5;</string>
            </property>
            <property name="text">
             <string>ITEM</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="asset_count_lbl">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>23</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel {color:#FFFFFF; padding-right: 15}</string>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ClickableQLabel" name="add_item_lbl">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Add Item</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="../resources.qrc">:/Resources/icon/plus-2-16.png</pixmap>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="ItemView" name="item_list_view">
          <property name="styleSheet">
           <string notr="true">QListView {color: #C2C2C2; font: bold 14px; width: 100px; background-color: #383838; border: 1px solid #3d3d3d;}
QListView::item:selected {background-color: #53728e;color: #C2C2C2;}
QListView:verticalScrollBar {background-color: #FFA91D;alternate-background-color: #FF0000;}</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_preview" native="true">
       <property name="maximumSize">
        <size>
         <width>320</width>
         <height>16777215</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin" stdset="0">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color:#cccccc;</string>
          </property>
          <property name="text">
           <string>PREVIEW</string>
          </property>
          <property name="margin">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="thumbnail_list_widget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>320</width>
            <height>360</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>320</width>
            <height>360</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #383838;</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="dragDropMode">
           <enum>QAbstractItemView::NoDragDrop</enum>
          </property>
          <property name="movement">
           <enum>QListView::Static</enum>
          </property>
          <property name="viewMode">
           <enum>QListView::IconMode</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="topMargin">
           <number>5</number>
          </property>
          <item>
           <widget class="QLabel" name="fileSummaryLabel">
            <property name="font">
             <font>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color:#cccccc;</string>
            </property>
            <property name="text">
             <string>FILE SUMMARY</string>
            </property>
            <property name="margin">
             <number>5</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="fileSummaryGridLayout">
          <item row="5" column="0" colspan="2">
           <widget class="QLabel" name="tagLabel">
            <property name="styleSheet">
             <string notr="true">color:#cccccc; background-color: #383838;</string>
            </property>
            <property name="text">
             <string>Tag:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QLabel" name="filePathLabel">
            <property name="styleSheet">
             <string notr="true">color:#cccccc; background-color: #383838;</string>
            </property>
            <property name="text">
             <string>/136arch/bds/asset/env/factory</string>
            </property>
            <property name="indent">
             <number>16</number>
            </property>
           </widget>
          </item>
          <item row="7" column="0" colspan="2">
           <widget class="QPlainTextEdit" name="commentText">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>100</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color:#cccccc; background-color: #383838;</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="plainText">
             <string>PEA /
안개속 멀리 보이는 폐공장 전경
(모팩)</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QLabel" name="assetNameLabel">
            <property name="styleSheet">
             <string notr="true">color:#cccccc; background-color: #383838;</string>
            </property>
            <property name="text">
             <string>factory</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="uploadUserLabel">
            <property name="styleSheet">
             <string notr="true">color:#cccccc; background-color: #383838;</string>
            </property>
            <property name="text">
             <string>Uploaded by rndvfx</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QLabel" name="uploadTimeLabel">
            <property name="styleSheet">
             <string notr="true">color:#cccccc; background-color: #383838;</string>
            </property>
            <property name="text">
             <string>2021-02-25 10:46:43</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="6" column="0" colspan="2">
           <widget class="QListWidget" name="tag_list_widget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>80</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QListWidget {
	padding: 2px;
	background-color: #383838;
}

QListWidget::item {
	border: 1px solid #518e55;
	border-radius: 3px;
	background-color: #3b5045;
	margin: 2px;
	color: white;
}

QListWidget::item::hover {
	background-color: #4d8453;
}</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="dragDropMode">
             <enum>QAbstractItemView::NoDragDrop</enum>
            </property>
            <property name="movement">
             <enum>QListView::Static</enum>
            </property>
            <property name="viewMode">
             <enum>QListView::IconMode</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="fileSizeLabel">
            <property name="styleSheet">
             <string notr="true">color:#cccccc; background-color: #383838;</string>
            </property>
            <property name="text">
             <string>Size: 2.2G</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="statusLabel_2">
            <property name="styleSheet">
             <string notr="true">color:#cccccc; background-color: #383838;</string>
            </property>
            <property name="text">
             <string>Status: Approved</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="topMargin">
           <number>5</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color:#cccccc;</string>
            </property>
            <property name="text">
             <string>BOOKMARK</string>
            </property>
            <property name="margin">
             <number>5</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="ClickableQLabel" name="reload_bookmark_lbl">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Reload</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="../resources.qrc">:/Resources/icon/reload.png</pixmap>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="BookmarkView" name="bookmark_list_view">
          <property name="styleSheet">
           <string notr="true">color: #C2C2C2; font: 12px; background-color: #383838</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="category_count_lbl">
       <property name="styleSheet">
        <string notr="true">color:white; font:10px;</string>
       </property>
       <property name="text">
        <string>progress....</string>
       </property>
       <property name="margin">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="statusLabel">
       <property name="styleSheet">
        <string notr="true">color:white; font:10px;</string>
       </property>
       <property name="text">
        <string>progress....</string>
       </property>
       <property name="margin">
        <number>5</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickableQLabel</class>
   <extends>QLabel</extends>
   <header>libs/clickableQLabel.h</header>
  </customwidget>
  <customwidget>
   <class>CategoryTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>customWidget/Category.h</header>
  </customwidget>
  <customwidget>
   <class>ItemView</class>
   <extends>QListView</extends>
   <header>customWidget/Item.h</header>
  </customwidget>
  <customwidget>
   <class>BookmarkView</class>
   <extends>QListView</extends>
   <header>customWidget/Bookmark.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>db1RadioButton</tabstop>
  <tabstop>db2RadioButton</tabstop>
  <tabstop>search_combobox</tabstop>
  <tabstop>searchEdit</tabstop>
  <tabstop>category_tree_widget</tabstop>
  <tabstop>item_list_view</tabstop>
  <tabstop>thumbnail_list_widget</tabstop>
  <tabstop>tag_list_widget</tabstop>
  <tabstop>commentText</tabstop>
  <tabstop>bookmark_list_view</tabstop>
 </tabstops>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
